src/qutip_qip/circuit.py

```diff
@@ -1486,6 +1497,13 @@ def _propagators_no_expand(self):
 
         return U_list
 
+    def compute_unitary(self):
+        """Evaluates all the gates in the quantum circuit of decomposed gates.
+        """
```

Comment:
We should document what is returned (i.e. the type of the return and a one line description).